src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx

```diff
@@ -104,6 +106,18 @@ export default class PatientViewPathwayMapper extends React.Component<
         if (!this.PathwayMapperComponent) {
             return null;
         }
+
+        var sampleIconData = {
+            sampleIndex: {},
+            sampleColors: {},
+        };
+        if (this.props.sampleManager) {
+            sampleIconData = {
+                sampleIndex: this.props.sampleManager.sampleIndex,
+                sampleColors: this.props.sampleManager.sampleColors,
+            };
+        }
```

Comment:
Never use `var`, either use `let` or `const`. Let's make this a `const` this way:

```suggestion
        const sampleIconData = this.props.sampleManager ? {
                sampleIndex: this.props.sampleManager.sampleIndex,
                sampleColors: this.props.sampleManager.sampleColors,
            }: {
                sampleIndex: {},
                sampleColors: {},
            };
```